tests/unit/test-cmd/test.build.js

```diff
@@ -72,6 +72,21 @@ describe('build', () => {
     });
   });
 
+  it('gives the correct custom name to an extension', () => {
```

Comment:
Let's also unit test the new helper method that does just interpolate the template string given the manifest data object (so that we can easily cover more of the other scenarios, like the error scenarios where the helper method is meant to throw).